app/models/ingredient.rb

```diff
@@ -1,3 +1,4 @@
 class Ingredient < ApplicationRecord
+   belongs_to :recipe
    validates :name, :cost, presence: true
 end
```

Comment:
Remember that when adding a `belongs_to` relationship, it's because there is a foreign key on that table. I do not recall there being a foreign key for a recipe on the ingredients table.

The tests for the association were already created, all you needed to do was add those associations in their respective models to get all the tests to pass.